server/controllers/donations.js

```diff
@@ -123,11 +105,12 @@ export {stripeDonation as stripe};
 export {stripeDonation as post};
 
 
+/*
 const paypalDonation = (req, res, next) => {
   const { group } = req;
   const { payment } = req.required;
   const currency = payment.currency || group.currency;
-  const amountFloat = payment.amount; // TODO: clean this up when we switch all amounts to INTEGER
+  const amount = parseInt(payment.amount * 100, 10);
```

Comment:
that's where having a very explicit `amountInCents` would make this more readable.

Should the frontend keep using float?